docs/openid.md

```diff
@@ -390,9 +390,6 @@ oidc_providers:
 
 ### Facebook
 
-Like Github, Facebook provide a custom OAuth2 API rather than an OIDC-compliant
-one so requires a little more configuration.
```

Comment:
it is nearly oidc-compliant now; the difference is spelt out more clearly in the text about discovery at the end.